src/netex-convertor/sharedHelpers.ts

```diff
@@ -216,73 +217,81 @@ export const replaceIWBusCoNocCode = (nocCode: string): string => {
 
 export const getCoreData = (
     operators: Operator[],
-    matchingData: PointToPointTicket | PeriodTicket | SchemeOperatorTicket,
+    ticket: PointToPointTicket | PeriodTicket | SchemeOperatorTicket,
 ): CoreData => {
-    const pointToPoint: boolean = matchingData.type === 'single' || matchingData.type === 'return';
+    const pointToPoint: boolean = ticket.type === 'single' || ticket.type === 'return';
 
     if (pointToPoint) {
-        const pointToPointMatchingData: PointToPointTicket = matchingData as PointToPointTicket;
+        const pointToPointTicket: PointToPointTicket = ticket as PointToPointTicket;
 
         return {
             opIdNocFormat: `noc:${operators[0].opId}`,
-            nocCodeFormat: `noc:${pointToPointMatchingData.nocCode}`,
+            nocCode: pointToPointTicket.nocCode,
+            nocCodeFormat: `noc:${pointToPointTicket.nocCode}`,
             currentDate: new Date(Date.now()),
             website: getCleanWebsite(operators[0].website),
-            brandingId: `op:${pointToPointMatchingData.nocCode}@brand`,
-            operatorIdentifier: pointToPointMatchingData.nocCode,
+            brandingId: `op:${pointToPointTicket.nocCode}@brand`,
+            operatorIdentifier: pointToPointTicket.nocCode,
             baseOperatorInfo: [],
             placeholderGroupOfProductsName: '',
             ticketUserConcat: '',
-            operatorPublicNameLineNameFormat: `${operators[0].operatorPublicName} ${pointToPointMatchingData.lineName}`,
-            nocCodeLineNameFormat: `${pointToPointMatchingData.nocCode}_${pointToPointMatchingData.lineName}`,
-            lineIdName: `Line_${pointToPointMatchingData.lineName}`,
-            lineName: pointToPointMatchingData.lineName,
+            operatorPublicNameLineNameFormat: `${operators[0].operatorPublicName} ${pointToPointTicket.lineName}`,
+            nocCodeLineNameFormat: `${pointToPointTicket.nocCode}_${pointToPointTicket.lineName}`,
+            lineIdName: `Line_${pointToPointTicket.lineName}`,
+            lineName: pointToPointTicket.lineName,
             isGeoZone: false,
             isMultiProduct: false,
             isMultiOperator: false,
             isPointToPoint: pointToPoint,
-            type: matchingData.type,
+            type: ticket.type,
```

Comment:
Just to be consistent, is there any reason this isn't `pointToPointTicket.type`?

Reply:
Changed.